regulations/templates/regulations/macros/nested.html

```diff
@@ -0,0 +1 @@
+{% extends template %}
```

Comment:
The existence of this template feels a little clunky -- if someone's trying to follow the code, they'd have to jump to an effectively empty file. Would it make sense to just render the template in `render_nested.py`?

One-liner:
```python
return loader.get_template(template).render(Context(context))
```